dynamic_graph_service/gsl_client/src/main/java/org/aliyun/gsl_client/Query.java

```diff
@@ -29,9 +29,9 @@ public void setId(String id) {
     this.id = id;
   }
 
-  public void feed(String alias, DataSource source) throws UserException {
-    PlanNode node = plan.getNode(alias);
-    node.setSource(source);
+  public void feed(DataSource source) throws UserException {
```

Comment:
why del alias?

Reply:
We only support feed input for source op now.